### DISCUSSION TITLE
Ignore leading zero on Australian phone numbers

### DISCUSSION THREAD

**Discussion by authentified-dave** [clerk/javascript#3098] [Feature Requests]
Hello Team Clerk,

This is a minor UX niggle. 

In Australia, our mobile numbers start with 0 and have the format 0XXX XXX XXX. When using this format on other services that have mobile phone number inputs with a region selector, they tend to either ignore/forgive/or strip silently the leading zero. 

Clerk however throws an error. 

<img width="388" alt="Screenshot 2024-04-03 at 5 36 29 pm" src="https://github.com/clerk/javascript/assets/159421700/47a86e84-3097-46b2-bb13-5149abad26b6">

Most Australians are used to typing their full number / not used to using international prefixes (but that's changing)

**Comment by jescalan** [comment]
https://github.com/orgs/clerk/discussions/3829
